src/test/e2e/pageobjects/SignIn.page.ts

```diff
@@ -0,0 +1,44 @@
+import { CommonPage } from './Common.page';
```

Comment:
we want all page objects for e2e tests to be in PageObjects directory not pageobjects.
This applies to all files which have been renamed without changes as well

Reply:
changed to PageObjects